include/alpaka/atomic/AtomicCudaBuiltIn.hpp

```diff
@@ -793,6 +793,50 @@ namespace alpaka
                     return atomicCAS(addr, compare, value);
                 }
             };
+
+            //#############################################################################
+            //! The GPU CUDA accelerator atomic operation.
+            template<
+                typename TOp,
+                typename T,
+                typename THierarchy>
+            struct AtomicOp<
+                TOp,
+                atomic::AtomicCudaBuiltIn,
+                T,
+                THierarchy>
+            {
+                //-----------------------------------------------------------------------------
+                ALPAKA_FN_HOST static auto atomicOp(
```

Comment:
@BenjaminW3 sorry for commenting on a merged PR, but I guess these default operations should be annotated with `__device__`

Reply:
yes you are right. I already fixed this in my follow-up branch